dashboard/src/components/Issue/IssueSection.tsx

```diff
@@ -0,0 +1,66 @@
+import { useMemo } from 'react';
+
+import { FaRegCircle, FaRegDotCircle } from 'react-icons/fa';
+
+import { FormattedMessage } from 'react-intl';
+
+import { UseQueryResult } from '@tanstack/react-query';
+
+import { Link } from '@tanstack/react-router';
+
+import { Skeleton } from '@/components/Skeleton';
+import ListingItem from '@/components/ListingItem/ListingItem';
+import { TIssue } from '@/types/general';
+
+import NoIssueFound from './NoIssueFound';
+
+const IssueSection = ({
+  data,
+  isLoading,
+  error,
+}: UseQueryResult<TIssue[]>): JSX.Element => {
+  const issueList = useMemo(
```

Comment:
Please if possible avoid this useMemo patter, just create another component with `memo`